src/storage/storage.service.ts

```diff
@@ -80,20 +82,28 @@ export class StorageService implements OnModuleInit, OnModuleDestroy {
 
   async saveRoleAssociation(party: string, sender: string, data: Role): Promise<void> {
     const roles = await this.storage.getObject(`lto:roles:${party}`).catch(() => { return {} });
-    
+
     roles[data.role] = { sender, type: data.type };
 
     return this.storage.setObject(`lto:roles:${party}`, roles);
   }
 
-  async saveAssociation(transaction: Transaction) {
+  async saveAssociation(transaction: Transaction): Promise<void> {
+    if (this.config.isAssociationGraphEnabled()) {
```

Comment:
Read the config in the constructor and set a `graphEnabled` property

Reply:
Done. did `onModuleInit` instead of constructor because it's easier on testing. Couldn't find a way to change the value before the constructor ran